samples/highcharts/css/color-count/demo.js

```diff
@@ -0,0 +1,68 @@
+Highcharts.chart('container', {
+    chart: {
+        styledMode: true,
+        // lower this value to see that colors for higher indices
```

Comment:
Most comments in demos (and in the repo) are capitalized so this one should too.